.circleci/config.yml

```diff
@@ -237,15 +246,20 @@ jobs:
     steps:
       - checkout
       - trust-github-key
-
+      - update-carthage-integration-commit
       # Carthage
       - restore_cache:
           keys: 
             - carthage-cache-{{ checksum "Cartfile.resolved" }}
       - run:
           name: Carthage Update
           working_directory: IntegrationTests/CarthageIntegration/
-          command: ./carthage.sh update --cache-builds
+          # install without building, then remove the tests and build, so that carthage
+          # doesn't try to build the other integration tests
+          command: |
+              ./carthage.sh update --no-build
+              rm -rf Carthage/Checkouts/purchases-root/IntegrationTests/
```

Comment:
Isn't this also removing `Carthage/Checkouts/purchases-root/IntegrationTests/CarthageIntegration`?

Reply:
yeah, but that's kinda intentional - it's only removing it from the _cloned_ version of the repo, which isn't needed.